src/OSPSuite.Infrastructure.Import/Core/DataSourceFileReaders/ExcelReader.cs

```diff
@@ -84,6 +84,21 @@ public bool MoveToNextRow()
          return true;
       }
 
+      public IEnumerable<string> RetrieveExcelSheets(bool excludeEmptySheets)
+      {
+         for (var i = 0; i < _book.NumberOfSheets; i++)
+         {
+            if (!excludeEmptySheets || !isSheetEmpty(_book.GetSheetAt(i)))
+               yield return _book.GetSheetName(i);
+         }
+      }
+
+      private bool isSheetEmpty(ISheet sheet)
+      {
+
```

Comment:
can you run resharper please to style this file?

Reply:
done. (it removed a few empty lines mainly)